src/lib/Faucet.sol

```diff
@@ -52,9 +53,7 @@ contract Faucet {
     }
 
     function canWithdraw(address _to) public view returns (bool, string memory, uint) {
-        if (!allowedAddresses[_to]) {
-            return (false, "Address not allowed to withdraw", 0);
-        }
+        require(isAddressAllowed(_to), "ADDRESS_NOT_WHITELISTED");
```

Comment:
`ADDRESS_NOT_WHITELISTED` can be made a constant. That way in case it ever needs to be changed, we will only have to change it in one place